remote-access/ssh/passwordless.md

```diff
@@ -84,6 +84,25 @@ If this does not work, you can get assistance on the [Raspberry Pi forums](https
 
 **Note:** you can also send files over SSH using the `scp` command (secure copy). See the [SCP guide](scp.md) for more information.
 
+## Adjust permissions for your home and .ssh directories
+
+If you can't establish a connection after following the steps above there might be a problem with your directory permissions. First, you want to check the logs for any errors:
+
+```bash
+tail -f /var/log/secure
+# might return:
+Nov 23 12:31:26 raspberrypi sshd[9146]: Authentication refused: bad ownership or modes for directory /home/pi
+```
+
+If the log says `Authentication refused: bad ownership or modes for directory /home/pi` there is a permission problem regarding your home directory. SSH needs your home and `~/.ssh` directory to not have group write access. You can adjust the permissions using `chmod` (make sure to replace `user` with your username, e.g. `pi`):
+
+```bash
+chmod g-w /home/user
+chmod 700 /home/user/.ssh
```

Comment:
Given that the documentation here specifically targets Raspberry Pi OS, perhaps it makes more sense to use `/home/pi/.ssh` etc. in the instructions rather than `/home/user/.ssh` ?
Alternatively, I think you could just use `$HOME/.ssh` ? :shrug:

Reply:
Sure, using `$HOME` sounds good. I like that there's no loss in generality as opposed to using `/home/pi/` directly.